public/app/core/components/Select/MetricSelect.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import _ from 'lodash';
+
+import Select from './Select';
+import { Variable } from 'app/types/templates';
+
+export interface Props {
+  onChange: (value: string) => void;
+  options: any[];
```

Comment:
This should probably be typed the same as the https://github.com/grafana/grafana/blob/05d826bdcb685c78bce1e37588b76c33d2c7cd95/public/app/core/components/Select/Select.tsx#L16